frontend/appflowy_tauri/src/appflowy_app/utils/slate/hotkey.ts

```diff
@@ -14,9 +16,72 @@ const HOTKEYS: Record<string, string> = {
 export function triggerHotkey(event: React.KeyboardEvent<HTMLDivElement>, editor: Editor) {
   for (const hotkey in HOTKEYS) {
     if (isHotkey(hotkey, event)) {
```

Comment:
same as the previous PR' comment.

I prefer using find to reduce the number of {} in the code.

```ts
const hotkey = HOTKEYS.find(...);
if (hotkey) {
  // ...
}
```

Reply:
It is object, not array